addon/components/link-to.ts

```diff
@@ -820,14 +820,12 @@ const LinkComponent = EmberComponent.extend({
   ) {
     let { _routing: routing } = this;
 
-    return () => {
-      payload.transition = routing.transitionTo(
-        qualifiedRouteName,
-        models,
-        queryParams,
-        shouldReplace
-      );
-    };
+    payload.transition = routing.transitionTo(
```

Comment:
this function had to return callback when was wrapped into `flaggedInstrument` but now we just need to invoke `transitionTo`